src/index.js

```diff
@@ -1,5 +1,68 @@
 import 'bootstrap';
 import './scss/index.scss';
-import displayScoreArr from './js/displayLB.js';
+import tableData from './js/displayLB.js';
 
-displayScoreArr();
\ No newline at end of file
+const scoreForm = document.querySelector('#score-form');
+
+const getData = () => {
+  fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/VwaMYCtnuP7TunvNhm2w/scores/',
+    {
+      method: 'GET',
+    },
+  )
+    .then((res) => res.json())
+    .then((final) => {
+      const data = final.result;
+      data.forEach((val) => {
+        tableData(val.user, val.score);
+      });
+    });
+};
+
+document.addEventListener('DOMContentLoaded', () => {
+  getData();
+});
+
+// Refresh page
+document.getElementById('getter').addEventListener('click', () => {
+  getData();
+  window.location.reload(true); // Refresh browser
+});
+
+scoreForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const formName = document.querySelector('#lb-name');
+  const formScore = document.querySelector('#lb-score');
+
+  const alertBox = document.querySelector('#alert');
+
+  const formObject = {
+    user: formName.value,
+    score: formScore.value,
+  };
+
+  fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/VwaMYCtnuP7TunvNhm2w/scores/',
+    {
+      method: 'POST',
+      body: JSON.stringify(formObject),
+      headers: {
+        'Content-Type': 'application/json', // Specify the type of data you are sending
+      },
+    },
+  ) // Post data to the API
+  // Actions after posting the data
+    .then((res) => res.json())
+    .then((final) => {
+      formName.value = '';
+      formScore.value = '';
+      alertBox.innerHTML = final.result;
+      alertBox.style.opacity = 1;
+      alertBox.style.PointerEvents = 'all';
+      setTimeout(() => {
+        alertBox.style.opacity = 0;
+        alertBox.style.PointerEvents = 'none';
+      }, 2500);
+    });
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻  However According to the project requirement, you should Use `async` and `await` with the `fetch` method to send and receive data but you used promises instead 

- Syntax: Following is the syntax which we could use in order to execute an asynchronous method async-await:

```
let method = async () => {
      let response = await fetch("https://.....");
      let data = await response.json();
      console.log(data);
    .....
}
```

- For more documentation you can check  [Promises vs Async Await](https://www.loginradius.com/blog/engineering/callback-vs-promises-vs-async-await/)

Reply:
not resolved

Reply:
yes, it is, Sorry I don't have much time for Know, but I'm working on it on the weekend.

Reply:
Even if the previous commit was is **optional** , I know I should change it soon.